libs/state/selections/src/lib/accumulation-observable.ts

```diff
@@ -30,28 +30,35 @@ export function createAccumulationObservable<T extends object>(
   stateSlices = new Subject<Partial<T>>(),
   accumulatorObservable = new BehaviorSubject(defaultAccumulator)
 ): Accumulator<T> {
-  const signal$ = merge(
-    stateObservables.pipe(
-      distinctUntilChanged(),
-      mergeAll(),
-      observeOn(queueScheduler)
+  const signal$ = connectable(
+    merge(
+      stateObservables.pipe(
+        distinctUntilChanged(),
+        mergeAll(),
+        observeOn(queueScheduler)
+      ),
+      stateSlices.pipe(observeOn(queueScheduler))
+    ).pipe(
+      withLatestFrom(accumulatorObservable.pipe(observeOn(queueScheduler))),
+      scan(
+        (state, [slice, stateAccumulator]) => stateAccumulator(state, slice),
+        {} as T
+      ),
+      tap({
+        next: (newState) => (compositionObservable.state = newState),
+        error: (error) => console.error(error),
+      }),
+      catchError((e) => EMPTY)
     ),
-    stateSlices.pipe(observeOn(queueScheduler))
-  ).pipe(
-    withLatestFrom(accumulatorObservable.pipe(observeOn(queueScheduler))),
-    scan(
-      (state, [slice, stateAccumulator]) => stateAccumulator(state, slice),
-      {} as T
-    ),
-    tap(
-      (newState) => (compositionObservable.state = newState),
-      (error) => console.error(error)
-    ),
-    // @Notice We catch the error here as it get lost in between `publish` and `publishReplay`. We return empty to
-    catchError((e) => EMPTY),
-    publish()
+    {
+      connector: () => new Subject<T>(),
+      resetOnDisconnect: false,
```

Comment:
is this really what we want?

> If false, upon disconnection, the connecting subject will remain the
connecting subject, meaning the resulting observable will not go "cold" again,
and subsequent repeats or resubscriptions will resubscribe to that same subject.